### ISSUE TITLE
Unwavering Stance does not remove chance to evade

### ISSUE THREAD

**Issue by iansjk** [Openarl/PathOfBuilding#126]
Even if Unwavering Stance is allocated in the passive tree I have a non-zero evasion rating on the stats sheet. Also, if you go to the passive tree and enable "Show node power", nodes that grant only evasion rating are still shown as having defensive power even though US is allocated.

**Comment by iansjk** [comment]
Oh, reading the wiki more closely this isn't really a bug:

> Unwavering Stance does not modify evasion **rating**. This is significant mainly because of the Iron Reflexes keystone, which converts evasion rating to armour rating.

It's a bit unintuitive though that it considers evasion rating to have defensive node power when chance to evade is always 0% and I don't have Iron Reflexes allocated.

**Comment by Openarl** [comment]
Perhaps, but it's a minor issue; I don't think it's worth tinkering with the defence power calculation just to prevent it.

**Comment by SparkusK** [comment]
I just tinkered with this on the current version - even if I've got _both_ Iron Reflexes _and_ Unwavering Stance, there are still bright blue nodes that only give Evasion-related bonuses. 

It's not a minor issue. The Node Power thing doesn't work if it's inaccurate. It's there so I can get an at-a-glance indication of where I need to be heading on the tree, and if it's misleading me with inaccuracies then that means it's borked.
